Source/Dafny/TestGenerationOptions.cs

```diff
@@ -60,23 +60,28 @@ public enum Modes { None, Block, Path };
 
     public string Help => @"
 /generateTestMode:<None|Block|Path>
-    None is the default and has no effect.
-    Block prints block-coverage tests for the given program.
-    Path prints path-coverage tests for the given program.
-    Using /definiteAssignment:3 and /loopUnroll is highly recommended when
-    generating tests.
+    None (default) - Has no effect.
+    Block - Prints block-coverage tests for the given program.
+    Path - Prints path-coverage tests for the given program.
+
+    Using /definiteAssignment:3 and /loopUnroll is highly recommended
+    when generating tests.
+
 /warnDeadCode
     Use block-coverage tests to warn about potential dead code.
+
 /generateTestSeqLengthLimit:<n>
-    If /testMode is not None, using this argument adds an axiom that sets the
-    length of all sequences to be no greater than <n>. This is useful in
-    conjunction with loop unrolling.
+    If /testMode is not None, using this argument adds an axiom that
+    sets the length of all sequences to be no greater than <n>. This is
+    useful in conjunction with loop unrolling.
+
 /generateTestTargetMethod:<methodName>
     If specified, only this method will be tested.
+
 /generateTestInlineDepth:<n>
-    0 is the default. When used in conjunction with /testTargetMethod, this
-    argument specifies the depth up to which all non-tested methods should be
-    inlined.";
+    0 is the default. When used in conjunction with /testTargetMethod,
+    this argument specifies the depth up to which all non-tested methods
+    should be inlined.";
```

Comment:
```suggestion
    should be inlined.
";
```

Reply:
Seems there ought to be a newline for this string - but I suppose its lack is compensated for elsewhere